src/ray/raylet/worker_pool.cc

```diff
@@ -1470,7 +1468,37 @@ void WorkerPool::PrestartWorkers(const TaskSpecification &task_spec,
                    << backlog_size << " and available CPUs " << num_available_cpus
                    << " num idle workers " << state.idle.size()
                    << " num registered workers " << state.registered_workers.size();
-    PrestartDefaultCpuWorkers(task_spec.GetLanguage(), num_needed);
+    PrestartWorkersWithRuntimeEnv(task_spec, num_needed);
```

Comment:
I think we can unify `PrestartDefaultCpuWorkers` and `PrestartWorkersWithRuntimeEnv` into a single function:

```
PrestartWorkers(ray::Language language, int64_t num_needed, JobID job_id = JobID::Nil(),  int runtime_env_hash = 0,
      const std::string &serialized_runtime_env_context = "{}",
      const rpc::RuntimeEnvInfo &runtime_env_info = rpc::RuntimeEnvInfo(),) {
}
```

Reply:
I don't see any benefit, the code path looks completely different.

Reply:
Just to reduce round-trip, I updated the implementation as you wish